### ISSUE TITLE
Font

### ISSUE THREAD

**Issue by TanishSama** [rhenwinch/Flixclusive#76]
### Describe your suggested feature

Can you add another font options possibily? Like the font used in the Netflix app or website

**Comment by rhenwinch** [comment]
I'll consider it!

Looks like Netflix has their own font:
![image](https://github.com/rhenwinch/Flixclusive/assets/52451354/44c129a5-361a-4bab-83dc-b246eae69dd6)

**Comment by TanishSama** [comment]
Yes , it may give trouble, however there are many other fonts to choose from which are available for public use.

Maybe you can make a system to allow users to choose font files? That way one can use any font he wants

**Comment by rhenwinch** [comment]
> Maybe you can make a system to allow users to choose font files? That way one can use any font he wants

Hmm, fonts have families if you aren't aware of that. I don't have any idea yet how I could integrate a dynamic font family so users could choose their own fonts. However, I think what I can consider for now is let users choose between _system-used font_ or _app-used font_

What you think? Does it satisfy your concern?

**Comment by TanishSama** [comment]
Yes system default font is also a good option, thanks

**Comment by ZZ1234mm** [comment]
Add download feature in next ### _**version******_
